packages/python/src/components/Variable.tsx

```diff
@@ -0,0 +1,24 @@
+import { Children, code } from "@alloy-js/core";
+import { usePythonNamePolicy } from "../name-policy.js"; // assuming you have this
+
+export interface VariableProps {
```

Comment:
I think this should also take children like TS VarDeclaration. When you are providing complex initializers it is quite annoying to pass that initializer in as a property as opposed to just placing it in the children of Variable.